walkthroughs/1-integrating-event-and-api-driven-apps/walkthrough.adoc

```diff
@@ -67,10 +68,12 @@ image::images/arch.png[integration, role="integr8ly-img-responsive"]
 
 // tag::creating-api-connector[]
 
-
 [id='creating-api-connector_{context}']
 [.integr8ly-docs-header]
-=== Creating an API Connector 
+=== Creating an API Connector
+
+Fuse Online provides connectors for many services, for example, Gmail and IRC.
```

Comment:
Nice. This is simple but really helpful to give a broader idea of Fuse Online. This alternative jumped to mind for me "Fuse Online is an enterprise integration platform that provides connectors for many services, for example, Gmail and IRC. It can also be extended with custom connectors."

Reply:
And to be more specific we can state "Custom connectors can be an Open API definition (Swagger) or a JAR containing functionality."

Reply:
It can be extended with custom connectors, using either OpenAPI definitions (Swagger) or using JAR files.